README.md

```diff
@@ -594,8 +594,9 @@ config :my_app, MyApp.Repo,
 config :fun_with_flags, :persistence,
   adapter: FunWithFlags.Store.Persistent.Ecto,
   repo: MyApp.Repo,
-  ecto_table_name: "your_table_name" # optional
-  # Default table name is "fun_with_flags_toggles".
+  ecto_table_name: "your_table_name", # optional, defaults to "fun_with_flags_toggles"
+  ecto_primary_key_type: :binary_id # optional, defaults to :id
+  # For the primary key type, see also: https://hexdocs.pm/ecto/Ecto.Schema.html#module-schema-attributes
```

Comment:
I'd include the current version in the URL, so that if that changes the link won't break.
(It might become outdated, which is a different problem, but at least it will be clear that it is outdated.)

I've tried to use use the same approach elsewhere in the docs, albeit not consistently.

```suggestion
  # For the primary key type, see also: https://hexdocs.pm/ecto/3.10.3/Ecto.Schema.html#module-schema-attributes
```